internal/factsengine/gatherers/sbddump.go

```diff
@@ -0,0 +1,50 @@
+package gatherers
+
+import (
+	"fmt"
+	"strings"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/trento-project/agent/internal/utils"
+)
+
+const (
+	SBDDumpFactKey = "SBD_dump"
```

Comment:
By the way, in other gatherers I have renamed this to `*GathererName` as the `FactKey` was something older, and it doesn't make much sense.
It would be `SBDDumpGathererName`